README.md

```diff
@@ -5,7 +5,7 @@ This is the common backend for Tech-Ambit web and android application.
 # Directory structure
 ```` 
     .
-    |──rest_apis/
+    ├──rest_apis/
```

Comment:
We've updated the structure, right? That should be reflected here as well.

Reply:
Yeah I'll do this